### DISCUSSION TITLE
use automation in mantine

### DISCUSSION THREAD

**Discussion by ofirgo1** [mantinedev/mantine#6212] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

6

### What package has an issue?

@mantine/core

### What framework do you use?

create-react-app (CRA)

### In which browsers you can reproduce the issue?

All

### Describe the bug

hii,
i want to add automation to my web application and i got an issue.
in the select component of mantine we add the options in an array, so we can't add id to single option.
I would like to know if there is an option to solve this issue.

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by mmounirf** [comment]
You can use Combobox instead of Select if you want custom options. Here's an [example](https://mantine.dev/combobox/?e=SelectOptionComponent) of using custom options with Combobox
